### PR TITLE
Updated example in README.md for increased outreach

### LINKED ISSUE #80: getting this error in the default example from readme page
```python
# /// script
# dependencies = [
#   "mcp"
# ]
# ///
from mcp.server import Server, NotificationOptions
from mcp.server.models import InitializationOptions
import mcp.server.stdio
import mcp.types as types

# Create a server instance
server = Server("example-server")

# Add prompt capabilities
@server.list_prompts()
async def handle_list_prompts() -> list[types.Prompt]:
    return [
        types.Prompt(
            name="example-prompt",
            description="An example prompt template",
            arguments=[
                types.PromptArgument(
                    name="arg1",
                    description="Example argument",
                    required=True
                )
            ]
        )
    ]

@server.get_prompt()
async def handle_get_prompt(
    name: str,
    arguments: dict[str, str] | None
) -> types.GetPromptResult:
    if name != "example-prompt":
        raise ValueError(f"Unknown prompt: {name}")

    return types.GetPromptResult(
        description="Example prompt",
        messages=[
            types.PromptMessage(
                role="user",
                content=types.TextContent(
                    type="text",
                    text="Example prompt text"
                )
            )
        ]
    )

async def run():
    # Run the server as STDIO
    async with mcp.server.stdio.stdio_server() as (read_stream, write_stream):
        await server.run(
            read_stream,
            write_stream,
            InitializationOptions(
                server_name="example",
                server_version="0.1.0",
                capabilities=server.get_capabilities(
                    notification_options=NotificationOptions(),
                    experimental_capabilities={},
                )
            )
        )

if __name__ == "__main__":
    import asyncio
    asyncio.run(run())
```

Error :
```
--------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
Cell In[1], line 70
     68 if __name__ == "__main__":
     69     import asyncio
---> 70     asyncio.run(run())

File [/usr/lib/python3.12/asyncio/runners.py:190](http://localhost:8888/usr/lib/python3.12/asyncio/runners.py#line=189), in run(main, debug, loop_factory)
    161 """Execute the coroutine and return the result.
    162 
    163 This function runs the passed coroutine, taking care of
   (...)
    186     asyncio.run(main())
    187 """
    188 if events._get_running_loop() is not None:
    189     # fail fast with short traceback
--> 190     raise RuntimeError(
    191         "asyncio.run() cannot be called from a running event loop")
    193 with Runner(debug=debug, loop_factory=loop_factory) as runner:
    194     return runner.run(main)

RuntimeError: asyncio.run() cannot be called from a running event loop
```

### PR DESCRIPTION
**Pull request by miguelg719**
Updated the examples in README.md for improved accessibility and further outreach. 
Fixes #80 

## Motivation and Context

Took a while to get the first test server running while trying to build a new custom MCP server. Although I did not give up, I'm afraid other people might; especially during first contact. I'm just trying to make it more accessible.

## How Has This Been Tested?

Simplified running of the two proposed snippets into two separate files. As long as python is executable from the terminal, it will work. 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation update

## Checklist

- [x] I have read the [MCP Documentation](https://modelcontextprotocol.io)
- [x] My code follows the repository's style guidelines
- [x] New and existing tests pass locally
- [x] I have added appropriate error handling
- [x] I have added or updated documentation as needed

## Additional context

Thanks for your efforts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,39 +152,52 @@ if __name__ == "__main__":
...
+    args=["example_server.py"], # Optional command line arguments
     env=None # Optional environment variables
 )
 
-async with stdio_client(server_params) as (read, write):
-    async with ClientSession(read, write) as session:
-        # Initialize the connection
-        await session.initialize()
+async def run():
+    async with stdio_client(server_params) as (read, write):
```
Comment (logan-markewich) [inline README.md:169]: meanwhile I'm just trying to figure out how to launch a client that doesn't require launching a server in the same script lol

Comment (miguelg719) [reply README.md:169]: It seems like anyio supports tcp (connect_tcp). One option is to follow that route, having the server listen in a port and the client connect to it. That's probably the idea moving forward when running client/servers from different hosts (better if upgraded to TLS).

Comment (dsp-ant) [reply README.md:169]: @logan-markewich feel free to move that to a discussion https://github.com/orgs/modelcontextprotocol/discussions

**Review by dsp-ant — APPROVED** [review]
Thank you. These improvements make a lot of sense and help quite a bit. Really appreciated.

**Pull request merged**